aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -368,6 +363,57 @@ void dispatch_fp8_rowwise_kernel(
   }
 }
 
+void check_inputs(
+    at::Tensor a,
```

Comment:
nit: could probably use const Tensor&

Reply:
Sure! Though, technically, all functions in this file could take tensors by const reference (except `out`)...